### PR TITLE
#77: onboard cohere and octo

### PR DESCRIPTION
**Pull request by mkrueger12**
Updated the Cohere and OctoML to work with routers

### CODE REVIEW

File: pkg/providers/cohere/chat.go [hunk 1]
```diff
@@ -107,46 +107,49 @@ func (c *Client) doChatRequest(ctx context.Context, payload *ChatRequest) (*sche
 	// Build request payload
 	rawPayload, err := json.Marshal(payload)
 	if err != nil {
-		return nil, fmt.Errorf("unable to marshal cohere chat request payload: %w", err)
```
Comment (roma-glushko) [inline pkg/providers/cohere/chat.go:110]: @mkrueger12 this is cohere package and the change in this package replace "cohere" by "openai". 
Is this intentionally?

**Review by roma-glushko — APPROVED** [review]
Great 👍

File: pkg/providers/cohere/chat.go (hunk 1 above)
Comment (mkrueger12) [reply pkg/providers/cohere/chat.go]: fixed

**Comment by mkrueger12** [comment]
I'll come back to the codecov failing test

**Pull request merged**